test/tools/config/config.py

```diff
@@ -8,7 +8,7 @@
 class Config:
     def __init__(self):
         self.dir = dirname(abspath(__file__))
-        self.base = dirname(dirname(dirname(abspath(__file__))))
+        self.base = dirname(dirname(dirname(dirname(abspath(__file__)))))
```

Comment:
Maybe with `pathlib.Path` it's more readable:
```
from pathlib import Path
```

```suggestion
        self.base = str(Path(abspath(__file__)).parent.parent.parent.parent)
```

Reply:
In theory, `parents[4]` should work too, but then we have to check the length first

Reply:
> Maybe with `pathlib.Path` it's more readable

maybe, i will add that with my next pr in all the places we do this 👍